Commands/ImportGA4Reports.php

```diff
@@ -332,14 +332,18 @@ public static function validatePropertyID($propertyId)
             throw new \Exception("Invalid property ID, required format properties/{propertyID}");
         }
     }
-    public function checkIfCanProcess()
+    public function checkIfCanProcess($idSite)
     {
-        $nextAvailableAt = (int) Option::get(GoogleAnalyticsGA4QueryService::DELAY_OPTION_NAME);
+        if (!$idSite) {
+            return true; // New import, so it should be allowed
```

Comment:
I received the following warning while starting an import since this is returning a boolean instead of the expected array:
`GoogleAnalyticsImporter/Commands/ImportGA4Reports.php(81): Warning - Trying to access array offset on value of type bool`
```suggestion
            return ['canProcess' => \true]; // New import, so it should be allowed
```